openzeppelin/token/erc721/library/ERC721_base.cairo

```diff
@@ -6,18 +6,18 @@ from starkware.cairo.common.alloc import alloc
 from starkware.starknet.common.syscalls import get_caller_address
 from starkware.cairo.common.uint256 import Uint256, uint256_check
 
-from contracts.utils.safemath import (
+from openzeppelin.utils.safemath import (
```

Comment:
i'm not sure `safemath` is just an util. maybe if we have a `security/` folder for `pausable.cairo`, it makes sense to have this one there too